proposals/lifetimes-and-provenance.md

```diff
@@ -220,7 +220,7 @@ NOTE: Internally to the compiler, references (which are always pointer sized)
 are treated as a register-passable types.  This means they compose correctly
 with implicit borrow semantics, and you can even pass a reference `inout` if you
 want to.  Such a thing is a "mutable reference to a reference", allowing the
-callee to mutate the callers reference.
+called to mutate the callers reference.
```

Comment:
"callee" is correct. I suppose "called" also works if you instead say "the called function," but "callee" is more common and pairs with "caller."